### PR TITLE
Add a custom trust manager so that we can report spans to an untrusted Zipkin

### PR DESCRIPTION
**Pull request by worldtiki**
Add a custom trust manager so that we can report spans to an untrusted Zipkin.

### CODE REVIEW

**Comment by adriancole** [comment]
now, come on. what kindof zipkin is untrustworthy?

**Comment by worldtiki** [comment]
> now, come on. what kindof zipkin is untrustworthy?

You would not believe the amount of impersonators there are out there :x

File: src/main/java/com/hotels/service/tracing/zipkintohaystack/forwarders/zipkin/http/ZipkinForwarder.java [hunk 1]
```diff
@@ -37,18 +39,33 @@
...
+                    .maxRequests(maxInFlightRequests)
+                    .writeTimeout(writeTimeoutMillis)
+                    .compressionEnabled(compressionEnabled);
 
-        builder.clientBuilder()
-                .connectionPool(new ConnectionPool(maxIdleConnections, 5, TimeUnit.MINUTES))
-                .pingInterval(60, TimeUnit.SECONDS);
+            builder.clientBuilder()
+                    .connectionPool(new ConnectionPool(maxIdleConnections, 5, TimeUnit.MINUTES))
+                    .pingInterval(61, TimeUnit.SECONDS);
```
Comment (nikos912000) [inline src/main/java/com/hotels/service/tracing/zipkintohaystack/forwarders/zipkin/http/ZipkinForwarder.java]: Is there a reason behind the `pingInterval` change?

Comment (worldtiki) [reply src/main/java/com/hotels/service/tracing/zipkintohaystack/forwarders/zipkin/http/ZipkinForwarder.java]: No, this was done accidentally. Thanks for catching.

**Pull request merged**